assets/js/navigation.js

```diff
@@ -56,7 +56,11 @@
 
 		}
 
-		$this.off( 'hover', position );
+		$this.on( 'mouseleave', function() {
+
+			$( this ).children( '.sub-menu' ).first().removeAttr( 'style' );
```

Comment:
@EvanHerman Since `$submenu` is already defined, I think this can be changed to:

```js
$submenu.removeAttr( 'style' );
```